modules/cats-effect/src/main/scala/pureconfig/module/catseffect/package.scala

```diff
@@ -111,6 +147,25 @@ package object catseffect {
     pureconfig.saveConfigAsPropertyFile(conf, outputPath, overrideOutputPath, options)
   }
 
+  /**
+   * Save the given configuration into a property file
+   *
+   * @param conf The configuration to save
+   * @param outputPath Where to write the configuration
+   * @param blocker the blocking context which will be used to load the configuration.
+   * @param overrideOutputPath Override the path if it already exists
+   * @param options the config rendering options
+   * @return The return action will save out the supplied configuration upon invocation
+   */
+  def blockingSaveConfigAsPropertyFileF[F[_], A](
```

Comment:
No we can't, scalac prints an error because two overloaded methods can not have both default arguments.

I wanted to ask about this, we have four options:

+ Remove the defaults from the deprecated method _(probably this breaks code, so not an option)_.
+ Remove the defaults from the new method, but I guess this makes less pleasant to use this new method.
+ Do not add these two methods, since an user can easily do this: `blocker.blockOn(oldMethod)` but providing the method in the library seems nice.
+ Different names.

I opted for the last one, just to make it compile.
I am sorry, right now this is still WIP I pushed because I needed to switch computers.
I will tag you when I finish the changes, but since you already commented this, feel free to review the four alternatives.

Reply:
Makes sense, I forgot about the default params. These methods are rather old and haven't been updated in a long time. We should rethink what API we want to offer on the write path, but for now I think it's fine to just rename 👍